web/src/utils/__tests__/getTheBestPromotionForDate.js

```diff
@@ -0,0 +1,60 @@
+import startOfDay from 'date-fns/startOfDay';
+import addDays from 'date-fns/addDays';
+import endOfDay from 'date-fns/endOfDay';
+import { getTheBestPromotionForDate } from '../getTheBestPromotionForDate';
```

Comment:
Jako, że jest tam TS, to odpuściłem sobie sprawdzanie typów na wejściu. To jest ok?

Reply:
Jeśli masz na myśli testowanie, czy funkcja waliduje typ danych, które dostaje na wejściu to w tak, imo jest to ok. Gdy mamy cały projekt w TSie, z dobrymi typami to zazwyczaj nie potrzebujemy takiej walidacji (są jednak miejsca gdzie może być potrzebna - np. w miejscu gdzie wpadają dane z API - tutaj TS nie będzie nas pilnował).